packages/fe/components/cid-table.vue

```diff
@@ -1,39 +1,77 @@
 <template>
   <div class="cid-table">
 
-    <div
-      v-if="cidList && cidList.length"
-      class="toolbar">
+    <div class="toolbar">
+      <Searchbar
```

Comment:
Entire toolbar should be hidden if CID list is empty

<img width="807" alt="Screenshot 2023-03-07 at 1 21 09 PM" src="https://user-images.githubusercontent.com/11708190/223528891-aab41eed-cdc7-4754-a724-66ccc6a18064.png">